Cargo.lock

```diff
@@ -1147,6 +1212,15 @@ dependencies = [
  "libc",
 ]
 
+[[package]]
+name = "hermit-abi"
```

Comment:
What's using this, and can we remove it?  We don't need ABI support for an obscure unikernel.